core/src/main/java/tech/tablesaw/columns/Column.java

```diff
@@ -164,6 +164,10 @@ default RollingColumn rolling(final int windowSize) {
     return new RollingColumn(this, windowSize);
   }
 
+  default RollingColumn rolling(final int windowSize, final boolean center) {
```

Comment:
this could use a javadoc for people not familiar with pandas's `center`

Reply:
apologies if this is duplicating something i wrote elsewhere. I made a comment but don't see it now, so perhaps i didn't save it.

I think the javadoc for this method needs to incorporate a slightly shorter version of the explanation you wrote for the PR. The comment here isn't enough for anyone to really get what it does, but the explanation you wrote is very clear. If you can add that, I think we can resolve this and approve and merge the PR